### ISSUE TITLE
deepQN doesn't learn

### ISSUE THREAD

**Issue by shendiaomo** [mrkulk/deepQN_tensorflow#1]
I've tried this program on a Mac, the agent never got total score more than 10..

Is there some well-trained model as a demo?

**Comment by gliese581gg** [comment]
Yeah It never get score higher than 10~20 because I removed fixed target network which is essential to prevent oscilation and reach high score.
I'm working on new version  and it's almost done.
I'll upload that soon!

**Comment by shendiaomo** [comment]
Great news!

**Comment by shendiaomo** [comment]
It seems in the paper of 2015 there was no target network in the algorithm, and this code is based on this version. 

In the paper of 2016, they say that a target network is necessary. This is what you called the new version, isn't is?

But now I'm very curious how the deepmind people made the first version work...

**Comment by mrkulk** [comment]
target network is necessary for learning.

**Comment by ppwwyyxx** [comment]
If someone is interested, I have a [working implementation](https://github.com/ppwwyyxx/tensorpack/tree/master/examples/Atari2600) in tensorflow that can reach similar score (in the only game I've tested) to DeepMind's paper.
